### PR TITLE
Adding some functions to hook dictionary

### PR DESCRIPTION
**Pull request by timmywanttolearn**
Adding nn.LayerNorm, nn.InstanceNorm1d,nn.InstanceNorm2d,InstanceNorm3d, nn.PReLU, nn.Softmax to dictionary.

### CODE REVIEW

**Comment by Lyken17** [comment]
Please merge this pr with https://github.com/Lyken17/pytorch-OpCounter/pull/133
